.github/workflows/ci.yml

```diff
@@ -0,0 +1,93 @@
+name: CI
+
+on: [push]
```

Comment:
Let's build pull requests, too, so that we know whether the tests/build pass before merging.  Here's an example of how to do that: 

https://github.com/single-spa/single-spa-css/blob/ca53d0622f4ac582896bb58c0511914d0d9460ca/.github/workflows/build-and-test.yml#L3-L5

Reply:
One thing I noticed is you get a duplicate run when a PR opens since a `push` and `pull_request` will be triggered when the PR is opened.  Is that desirable?  

I do think people use both but box them to branch names - `main`, etc.  However, I think `push` is also sufficient on its own.  Happy to do whatever you think!

Reply:
`push` is insufficient because it doesn't run on pull requests originating from forks. I think it's better to do both `push` and `pull_request`. If you'd like to avoid the redundant build, you could try only having `push` enabled for the `main` branch.